src/styles/feedback.scss

```diff
@@ -34,7 +34,7 @@
   justify-content: start;
   font-size: 0.9rem;
   color: inherit;
-  border-color: var(--amplify-colors-brand-secondary-);
+  border-color: var(--amplify-colors-secondary-);
```

Comment:
This css custom property isn't doing anything so we can remove it, but actually looking at this CSS a little more we could remove all the base values on .vote-button so it becomes:
```
.vote-button {
  .amplify-text {
    padding-left: 7px;
  }
}
```
I think the base font size which is a teeny bit bigger is fine.